src/app.ts

```diff
@@ -547,16 +567,24 @@ yargs
         describe: 'The name of the profile to disable MFA'
       }),
     handler: async (args: { profile?: string }): Promise<void> => {
-      await disableMfa(args.profile);
+      try {
+        await disableMfa(args.profile);
+      } catch (error) {
+        console.error(chalk.red(error.message));
+      }
     }
   })
   .command({
     command: 'backup-config',
-    describe: 'Create a backup of your AWS and AWSX config files',
+    describe: 'Create a backup of your AWS and awsx config files',
```

Comment:
I think `AWSX` should be uppercase. I guess it's not really an acronym but still. What do you think?

Reply:
I was thinking lowercase because it's stylized like that everywhere else

Reply:
Yeah, okay, let's go with lowercase.